tests/phpunit/CRM/Utils/TokenConsistencyTest.php

```diff
@@ -565,13 +546,11 @@ public function getParticipantTokens(): array {
       '{participant.transferred_to_contact_id}' => 'Transferred to Contact ID',
       '{participant.role_id:label}' => 'Participant Role',
       '{participant.balance}' => 'Event Balance',
-      '{participant.' . $this->getCustomFieldName('participant_int') . '}' => 'Enter integer here :: participant_Group with field int',
+      '{participant.' . $this->getCustomFieldName('participant_int') . '}' => 'participant_Group with field int: Enter integer here',
```

Comment:
@demeritcowboy @colemanw I believe you thnk this order (which is also tecnically easier in this PR) is better?